Airport1/Airport1/Program.cs

```diff
@@ -0,0 +1,470 @@
+                    default: Console.WriteLine($"Program doesn't has operation type id {(int)operationType}"); break;
+                }
+            }
+        }
+
+        public static void AddFlight()
+        {
+            Console.WriteLine();
+            Console.WriteLine(@$"Enter flight type:
+{(int)FlightType.Arrival} - Arrival
+{(int)FlightType.Departure} - Departure");
+
+            var flightType = (FlightType)int.Parse(Console.ReadLine());
+
+            Console.WriteLine(@$"Enter flight date/time in format dd/mm/yy hh:mm");
+            var dateTime = DateTime.Parse(Console.ReadLine());
+
+            Console.WriteLine(@$"Enter flight city/port");
+            var cityPort = Console.ReadLine();
+
+            Console.WriteLine(@$"Enter airline");
+            var airline = Console.ReadLine();
+
+            Console.WriteLine(@$"Enter terminal");
+            Console.WriteLine(@"    Terminals:");
+            const int terminalCount = 26;
+            for (int i = 1; i <= terminalCount; i++)
+            {
+                var terminal = ((TerminalType)i).ToString();
+                Console.WriteLine(@$"    {i} = {terminal}");
+            }
+            var terminalType = (TerminalType)int.Parse(Console.ReadLine());
+
+            Console.WriteLine(@$"Enter flight status");
+            Console.WriteLine(@"    Flight Statuses:");
+            const int flightStatusesCount = 9;
+            for (int i = 1; i <= flightStatusesCount; i++)
+            {
+                var flightStatus = ((FlightStatusType)i).ToString();
+                Console.WriteLine(@$"    {i} = {flightStatus}");
+            }
+            var flightStatusType = (FlightStatusType)int.Parse(Console.ReadLine());
+
+            Console.WriteLine(@$"Enter gate:");
+            var gate = Console.ReadLine();
+
+            AppendFlight(flightType, dateTime, cityPort, airline, terminalType, flightStatusType, gate);
+            Console.WriteLine();
+        }
+
+        public static void ShowArrivals()
+        {
+            if (FlightsArray == null)
+            {
+                Console.WriteLine();
+                Console.ForegroundColor = ConsoleColor.Red;
+                Console.WriteLine("No flights");
+                Console.ForegroundColor = ConsoleColor.White;
+                Console.WriteLine();
+                return;
+            }
+
+            Console.WriteLine();
+            Console.WriteLine("Arrival flights:");
+
+            var found = false;
+            for (var i = 0; i < FlightsArray.GetLength(1); i++)
+            {
+                var flightType = (FlightType)FlightsArray[0, i];
+                if (flightType == FlightType.Arrival)
+                {
+                    found = true;
+                    PrintFlight(i);
+                }
+            }
+
+            if (!found)
+            {
+                Console.WriteLine("No arrival flights were found");
+            }
+            Console.WriteLine();
+        }
+
```

Comment:
Methods ShowArrivals() and ShowDepartures() are simsilar on 95%, maybe will be better do one method with parameters?

Reply:
Fixed db3b2ffa1708782e50b59fc22ad9887c552fbed5